lms/djangoapps/certificates/models.py

```diff
@@ -842,26 +842,38 @@ def course_key(self):
 class CertificateGenerationCourseSetting(TimeStampedModel):
     """Enable or disable certificate generation for a particular course.
 
-    This controls whether students are allowed to "self-generate"
+    DEPRECATED: 'enabled' controls whether students are allowed to "self-generate"
```

Comment:
Move these comments to the `help_text` kwargs on the fields. They are more useful there, where they will be exposed on the admin form, than in a class comment that (honestly) may not be read again.